mod-source-record-storage-server/src/main/java/org/folio/services/caches/ConsortiumConfigurationCache.java

```diff
@@ -0,0 +1,75 @@
+package org.folio.services.caches;
+
+import com.github.benmanes.caffeine.cache.AsyncCache;
+import com.github.benmanes.caffeine.cache.Caffeine;
+import io.vertx.core.Future;
+import io.vertx.core.Vertx;
+import io.vertx.core.http.HttpMethod;
+import io.vertx.core.json.JsonArray;
+import org.apache.http.HttpStatus;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.folio.dataimport.util.OkapiConnectionParams;
+import org.folio.dataimport.util.RestUtil;
+import org.folio.services.entities.ConsortiumConfiguration;
+import org.folio.services.exceptions.CacheLoadingException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import java.util.Optional;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.TimeUnit;
+
+@Component
+public class ConsortiumConfigurationCache {
```

Comment:
@RomanChernetskyi  Let's add some documentation for the new classes.

Reply:
Thanks, done